### ISSUE TITLE
Test card redesign for flu MVP 

### ISSUE THREAD

**Issue by lg-king** [CDCgov/prime-simplereport#5057]
### Description
(pair with dev)
We would like to add flu testing devices with minimal changes to the UI. Our learnings from this will be applied toward HIV pilot/expansion to other diseases. 

[Flu product brief](https://docs.google.com/document/d/1RO9A9z1wr5ATlMzs9JZhPfC29nlVZ8mZe32uAo4KSaw/edit?usp=sharing)

### MVP Scope
- Users can enter any flu LOINC from CA's list, along with associated test data, via the bulk uploader  
- The single entry submission will remain device-centric. We have LOINC and specimen mappings for 8 flu-only devices, so we will start with those. All of these devices have Flu A and Flu B results (positive, negative, inconclusive).

See "flu devices for single entry" tab - https://docs.google.com/spreadsheets/d/1dOh6_AWHwh4cvdXA4a1fPst1mox2RmcsvtIhhcM90I4/edit?usp=sharing 

### Out of scope for this iteration
- Changes to AoE / disease-specific AoE 
- Result values that are not positive or negative
- LOINC-centric test card design (i.e., result value for just one disease) 
- Positive-only test submissions

### Design considerations
- Test card design for a device with Flu A and Flu B result options (i.e., multiplex minus COVID) (consider widths of COVID, multiplex, and flu test cards)

**Comment by lg-king** [comment]
More information needed before we can proceed with design

**Comment by lg-king** [comment]
Laura King to reach out to CA on accepted specimen types.
